src/style/app.module.css

```diff
@@ -2995,1128 +2641,2457 @@ form > input {
+  background-color: white;
+  margin-top: 1rem;
+  padding: 20px;
+  border: 1px solid #dddddd;
+  border-radius: 10px;
+}
+
+.heading {
+  font-size: 1.1rem;
+}
+
+.donationInputContainer {
+  display: flex;
+  flex-direction: row;
+  margin-top: 20px;
+}
+
+.donateBtn {
+  padding-inline: 1rem !important;
+}
+
+.dropdow {
+  min-width: 6rem;
+}
+
+.inputArea {
+  border: none;
+  outline: none;
+  background-color: #f1f3f6;
+}
+
+.maxWidthDonate {
+  width: 100%;
+}
+
+.donateActions {
+  margin-top: 1rem;
+  width: 100%;
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.donationsContainer {
+  padding-top: 4rem;
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.donationCardsContainer {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  --bs-gutter-x: 0;
+}
+
+.colorLight {
+  background-color: #f5f5f5;
+}
+
+.content {
+  padding-top: 10px;
+  flex-grow: 1;
+}
+
+.modal-dialog {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  margin: 0;
+}
+
+.modal-content {
+  width: 100%;
+  max-width: 500px;
+}
+
+.modal-body {
+  text-align: center;
+}
+
+.modal-header,
+.modal-footer {
+  justify-content: center;
+  text-align: center;
+}
+
+.confirmation {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.contentOrg {
+  height: fit-content;
+  min-height: calc(100% - 40px);
+}
+
+.containerHeightOrg {
+  height: 100vh;
+  padding: 1rem 1.5rem 0 calc(300px + 1.5rem);
+}
+
+.colorPrimary {
+  background: #31bb6b;
+}
+
+.input {
+  margin-right: 10px;
+}
+
+.borderBox {
+  border: 1px solid #dddddd;
+}
+
+.borderRounded5 {
+  border-radius: 5px;
+}
+
+.borderRounded8 {
+  border-radius: 8px;
+}
+
+.borderRounded24 {
+  border-radius: 24px;
+}
+
+.topRadiusPeople {
+  border-top-left-radius: 24px;
+  border-top-right-radius: 24px;
+}
+
+.bottomRadius {
+  border-bottom-left-radius: 24px;
+  border-bottom-right-radius: 24px;
+}
+
+.shadow {
+  box-shadow: 5px 5px 4px 0px #31bb6b1f;
+}
+
+.colorGreen {
+  color: #31bb6b;
+}
+
+.maxWidthPeople {
+  max-width: 600px;
+}
+
+.mainContainerPeople {
+  width: 50%;
+  flex-grow: 3;
+}
+
+.contentPeople {
+  height: fit-content;
+  min-height: calc(100% - 40px);
+}
+
+.gap {
+  gap: 20px;
+}
+
+.greenBorder {
+  border: 1px solid #31bb6b;
+}
+
+.semiBold {
+  font-weight: 500;
+}
+
+.placeholderColor::placeholder {
+  color: #737373;
+}
+
+.btnsContainer {
+  display: flex;
+  margin: 1.5rem 0;
+}
+
+.btnsContainer .input {
+  flex: 1;
+  min-width: 18rem;
+  position: relative;
+}
+
+.btnsContainer input {
+  outline: 1px solid var(--bs-gray-400);
+  background-color: white;
+}
+
+.btnsContainer .input button {
+  width: 52px;
+}
+
+.titleContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 0.1rem;
+}
+
+.titleContainer h3 {
+  font-size: 1.25rem;
+  font-weight: 750;
+  color: #5e5e5e;
+  margin-top: 0.2rem;
+}
+
+.subContainer span {
+  font-size: 0.9rem;
+  margin-left: 0.5rem;
+  font-weight: lighter;
+  color: #707070;
```

Comment:
Use a variable. There are other similar colors defined at the top of the file you don't need to redefine one.